File: DoAnXepLich/backend/XepLichGiangDay.Api/Models/Entities/GiangVien.cs
namespace XepLichGiangDay.Api.Models.Entities;

public class GiangVien
{
    public int MaGiangVien { get; set; }
    public string HoTen { get; set; } = string.Empty;
    public string Email { get; set; } = string.Empty;
    public string? SoDienThoai { get; set; }
    public string? ChucDanh { get; set; }
    public int? MaBoMon { get; set; }
    public string TrangThai { get; set; } = "DangLamViec";
}

File: DoAnXepLich/frontend/src/types/giangVien.ts
export interface GiangVien {
  maGiangVien: number
  hoTen: string
  email: string
  soDienThoai?: string
  chucDanh?: string
  maBoMon?: number
  tenBoMon?: string
  trangThai: string
}

export interface TaoGiangVienDto {
  hoTen: string
  email: string
  soDienThoai?: string
  chucDanh?: string
  maBoMon?: number
  trangThai: string
}
